python/torch_mlir_e2e_test/configs/torchdynamo.py

```diff
@@ -8,6 +8,10 @@
 import numpy
 import torch
 import torch._dynamo as dynamo
+import torch._dynamo.config
+
+torch._dynamo.config.cache_size_limit = 6400
```

Comment:
Why is this needed?

Reply:
I don't know what the connection between `nopython=True` and their cache is but with this new settings that cache limit (of 64) is hit.

Reply:
I think the cache is always hit, even currently. We have one compilation per test case approximately which always overflows the cache.

Reply:
my issue must be because I ran with `-s` to identify the segfaults - will remove.